pkgdb2/static/pkgdb.css

```diff
@@ -156,6 +156,13 @@ a.wheel, a.wheel:hover, button.wheel, button.wheel:hover {
   background-position: 3px 50%;
 }
 
+label.active, label.active:hover {
+  background-image: url(https://admin.fedoraproject.org/pkgdb/static/arrow.png) !important;
```

Comment:
They are hard-coded for all the css, the only other option is to hard-code them as ``/pkgdb/static/...``.

We can't use flask's magic in the CSS files :(